src/analysis/processing/qgsalgorithmremovenullgeometry.cpp

```diff
@@ -74,6 +78,8 @@ QVariantMap QgsRemoveNullGeometryAlgorithm::processAlgorithm( const QVariantMap
   if ( !source )
     throw QgsProcessingException( invalidSourceError( parameters, QStringLiteral( "INPUT" ) ) );
 
+  const bool remove_empty = parameterAsBoolean( parameters, QStringLiteral( "REMOVE_EMPTY" ), context );
```

Comment:
```suggestion
  const bool removeEmpty = parameterAsBoolean( parameters, QStringLiteral( "REMOVE_EMPTY" ), context );
```

Reply:
Applied in https://github.com/qgis/QGIS/pull/35084/commits/3b6c04f70e57c2ce3b6bb2bee93c3f643f7a88b6